toolbox/gpu-operator/deploy_from_operatorhub.sh

```diff
@@ -34,16 +37,28 @@ if [[ "${1:-}" == "--help" || "${1:-}" == -h ]]; then
     exit 0
 fi
 
-if [[ "${1:-}" == "$DEPLOY_FROM_BUNDLE_FLAG" ]]; then
-    echo "Deploying the GPU Operator from OperatorHub using its master bundle."
+if [ "$#" -gt 1 ]; then
+    echo "FATAL: expected 0 or 1 parameter ... (got '$@')"
+    usage
+    exit 1
+elif [[ "${1:-}" == "$DEPLOY_FROM_BUNDLE_FLAG"* ]]; then
     ANSIBLE_OPTS="${ANSIBLE_OPTS} -e gpu_operator_deploy_from=bundle"
 
-    if [ "$#" -gt 1 ]; then
-        shift
-        echo "FATAL: $DEPLOY_FROM_BUNDLE_FLAG expects no additional parameter (got '$@')"
-        usage
-        exit 1
+    bundle=${1##$DEPLOY_FROM_BUNDLE_FLAG}
+    if [[ "$bundle" == "master" ]]; then
+        echo "Deploying the GPU Operator from OperatorHub using its master bundle."
+    else
+        echo "Deploying the GPU Operator from OperatorHub using bundle at $bundle."
+        ANSIBLE_OPTS="${ANSIBLE_OPTS} -e deploy_bundle_image=$bundle"
     fi
+
+elif [[ "${1:-}" == "$DEPLOY_ONLY_CLUSTERPOLICY_FLAG" ]]; then
+    ANSIBLE_OPTS="${ANSIBLE_OPTS} -e gpu_operator_deploy_from=pre-deployed"
+    echo "Not deploying the GPU Operator, only create the ClusterPolicy from the CSV."
+elif [[ "${1:-}" == "-"* ]]; then
```

Comment:
```suggestion
elif [[ "${1}" == "-"* ]]; then
```
There are a few uses of this syntax in this file. Doesn't the `:-` just substitute an empty string if $1 is not set -- making it unnecessary?

Reply:
It is necessary because of `set -u`

Reply:
Thanks for the explanation. The previous comments about setting the variables are still valid, right?

Reply:
I think they are, yeah

Reply:
yes, it's "if undefined or empty" (I didn't know the 'or empty' part :)
```
$ var=""
$ echo ${var:-default}
default

$ echo ${not_a_var:-default}
default
```